smash/fcore/operator/md_gr_operator.f90

```diff
@@ -277,6 +304,30 @@ subroutine gr5_time_step(setup, mesh, input_data, options, time_step, ac_mlt, ac
                 ! Transform from mm/dt to m3/s
                 ac_qt(k) = ac_qt(k)*1e-3_sp*mesh%dx(row, col)*mesh%dy(row, col)/setup%dt
 
+                !$AD start-exclude
+                !internal fluxes
+                if (returns%stats_flag) then
+                    returns%stats%internal_fluxes(row, col, 1) = pn
+                    returns%stats%internal_fluxes(row, col, 2) = en
+                    returns%stats%internal_fluxes(row, col, 3) = pr
+                    returns%stats%internal_fluxes(row, col, 4) = perc
+                    returns%stats%internal_fluxes(row, col, 5) = l
+                    returns%stats%internal_fluxes(row, col, 6) = prr
+                    returns%stats%internal_fluxes(row, col, 7) = prd
+                    returns%stats%internal_fluxes(row, col, 8) = qr
+                    returns%stats%internal_fluxes(row, col, 9) = qd
+                end if
+                if (returns%pn_flag) returns%pn(row, col, time_step) = pn
+                if (returns%en_flag) returns%en(row, col, time_step) = en
+                if (returns%pr_flag) returns%pr(row, col, time_step) = pr
+                if (returns%perc_flag) returns%perc(row, col, time_step) = perc
+                if (returns%lexc_flag) returns%lexc(row, col, time_step) = l
+                if (returns%prr_flag) returns%prr(row, col, time_step) = prr
+                if (returns%prd_flag) returns%prd(row, col, time_step) = prd
+                if (returns%qr_flag) returns%qr(row, col, time_step) = qr
+                if (returns%qd_flag) returns%qd(row, col, time_step) = qd
```

Comment:
ok for this first version. flux maps getter implemented?
Wouldn't this be better to use sthg like returns%internal_fluxes_maps(:,:,:,:) 
with 1..N fluxes which names is defined for each model structure (as for params or for stats%internal_fluxes above) in _constants.py?